syntax = "proto3";

package synchronizer;

import "google/protobuf/timestamp.proto";

// Coordinator Service Definition
service SynchService{
    rpc Change (Update) returns (Response) {}
}

message Update {
  int32 id = 1;
  int32 second_id = 2;
  repeated string arguments = 3;
}

message Response{
  string message = 1;
}